docs/user/ppl/admin/security.rst

```diff
@@ -34,7 +34,8 @@ Example: Create the ppl_role for test_user. then test_user could use PPL to quer
         ],
         "allowed_actions": [
           "indices:data/read/search*",
-          "indices:admin/mappings/get"
+          "indices:admin/mappings/get",
+          "indices:monitor/settings/get"
```

Comment:
could you explain more why this permission is required?

Reply:
okay I'll add to the doc.

I found out during testing with security plugin that this permission is needed. It's not directly related to this PR. It's a change to [get cluster max result window](https://github.com/opensearch-project/sql/blob/18661b4ed0a6f613ae91b052774751a0ea4f17c7/opensearch/src/main/java/org/opensearch/sql/opensearch/client/OpenSearchRestClient.java#L87) in a previous PR that needs this permission.

Reply:
I found during testing that this permission is needed. I believe it's [this part](https://github.com/opensearch-project/sql/blob/18661b4ed0a6f613ae91b052774751a0ea4f17c7/opensearch/src/main/java/org/opensearch/sql/opensearch/client/OpenSearchRestClient.java#L87) to get cluster max result window setting that needs this permission.

I shall remove this change and make a separate PR for it, as it's not directly related to this PR